content/docs/tutorials/gcp/gcp-ts-k8s-ruby-on-rails-postgresql.md

```diff
@@ -19,25 +21,25 @@ no_edit_this_page: true
 
 This example is a full end to end example of delivering a containerized Ruby on Rails application. It
 
-* Provisions a [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/) cluster
-* Provisions a fully managed Google Cloud SQL PostgreSQL database
-* Builds a containerized Ruby on Rails container image, and publishes it to Docker Hub
-* Deploys that container image as a Kubernetes Service inside of the provisioned GKE cluster
+-   Provisions a [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/) cluster
+-   Provisions a fully managed Google Cloud SQL PostgreSQL database
+-   Builds a containerized Ruby on Rails container image, and publishes it to Docker Hub
+-   Deploys that container image as a Kubernetes Service inside of the provisioned GKE cluster
 
 All of these happen behind a single `pulumi up` command, and are expressed in just a handful of TypeScript.
 
-# Prerequisites
+## Prerequisites
 
 Ensure you have [downloaded and installed the Pulumi CLI](https://www.pulumi.com/docs/get-started/install/).
-
+Esnure you have [downloaded and installed Docker](https://docs.docker.com/install/)
```

Comment:
Nit: Unrelated to this PR, but pointing it out since I noticed it: we should fix the `Ensure` typo in the examples repo.

Reply:
PR here: https://github.com/pulumi/examples/pull/526